lib/ssl/doc/src/using_ssl.xml

```diff
@@ -689,10 +721,6 @@ ssl:connect("localhost", 9999, [{verify, verify_peer},
     GET, can usually be regarded as safe but even they can be exploited by a large number of
     replays causing resource limit exhaustion and other similar problems.</p>
     <p>An example of sending early data with automatic and manual session ticket handling:</p>
-    <warning>
-    <p>The Early Data feature is experimental in this version of OTP.
-    </p>
-    </warning>
 
     <p><em>Server (with NSS key logging)</em></p>
```

Comment:
Section below no longer contains NSS key logging option, is the title still valid?

Reply:
Probably forgot to remove it, will fix!